ui/pages/asset/components/token-buttons.tsx

```diff
@@ -327,40 +329,39 @@ const TokenButtons = ({
         disabled={token.isERC721}
         tooltipRender={null}
       />
-      {isSwapsChain && (
```

Comment:
Instead of hiding the buttons, we are instead disabling them similar to other areas in the UI.